src/overlays/actors/ovl_En_Heishi4/z_en_heishi4.h

```diff
@@ -4,11 +4,37 @@
 #include <ultra64.h>
 #include <global.h>
 
+typedef enum {
+    /* 0x00 */ HEISHI4_KAKRIKO_ENTRANCE,
+    /* 0x04 */ HEISHI4_IMPA = 4,
+    /* 0x07 */ HEISHI4_DYING_GUARD = 7,
+    /* 0x08 */ HEISHI4_MARKET_NIGHT_GUARD,
+};
```

Comment:
This isn't necessarily the best but here are my suggestions to improve the consistency of this enum:
```suggestion
typedef enum {
    /* 0x00 */ HEISHI4_AT_KAKRIKO_ENTRANCE,
    /* 0x04 */ HEISHI4_AT_IMPAS_HOUSE = 4,
    /* 0x07 */ HEISHI4_AT_MARKET_NIGHT_DYING = 7,
    /* 0x08 */ HEISHI4_AT_MARKET_NIGHT
} Heishi4Type;
```
Also we usually have it below the struct but that's not an actual rule so I guess it's w/e